examples/index.json

```diff
@@ -550,15 +550,15 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Querying tables",
-                "url": "querying-tables",
+                "name": "tables querying",
```

Comment:
Why are we changing this?

Reply:
This is a title btw.

Reply:
Reverted the name tback o `Querying tables` and the URL to `querying-tables`.